doc/developer_guide/index.md

```diff
@@ -82,7 +109,7 @@ All available tasks can be found by running `pixi task list`, the following sect
 
 ### Editable install
 
-It can be advantageous to install the Panel in [editable mode](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs):
+It can be advantageous to install Panel in [editable mode](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs):
```

Comment:
Its still not fully clear to me when I want to do this/ when it must be done.

For example if I want to run `build-docs` do I then need to do an editable install to `docs` environment first? And why does it seem to work without doing this?

Reply:
Because presumably the docs env automatically adds an editable install. Please confirm @hoxbro.

Reply:
See the comment I made [here](https://github.com/holoviz/panel/pull/7121#discussion_r1713221651).

If you have more questions after reading that, I will happily answer them.